src/Schema/Comparator.php

```diff
@@ -20,31 +21,29 @@
  */
 class Comparator
 {
-    /**
-     * @return SchemaDiff
-     *
-     * @throws SchemaException
-     */
-    public static function compareSchemas(Schema $fromSchema, Schema $toSchema)
-    {
-        $c = new self();
-
-        return $c->compare($fromSchema, $toSchema);
-    }
-
     /**
      * Returns a SchemaDiff object containing the differences between the schemas $fromSchema and $toSchema.
      *
-     * The returned differences are returned in such a way that they contain the
-     * operations to change the schema stored in $fromSchema to the schema that is
-     * stored in $toSchema.
+     * This method is marked as static to avoid a BC break but should be called
+     * non-statically since it will not be static in the next major release.
      *
      * @return SchemaDiff
      *
      * @throws SchemaException
      */
-    public function compare(Schema $fromSchema, Schema $toSchema)
+    public static function compareSchemas(Schema $fromSchema, Schema $toSchema)
     {
+        /*
```

Comment:
Please remove this. Apparently, `$this` is unavailable in a `static` method call even if it's called non-statically.